federation/ldap/src/main/java/org/keycloak/storage/ldap/mappers/membership/group/GroupLDAPStorageMapperFactory.java

```diff
@@ -282,6 +292,12 @@ public void validateConfiguration(KeycloakSession session, RealmModel realm, Com
         }
 
         LDAPUtils.validateCustomLdapFilter(config.getConfig().getFirst(GroupMapperConfig.GROUPS_LDAP_FILTER));
+
+        checkMandatoryConfigAttribute(GroupMapperConfig.LDAP_GROUPS_PATH, "Groups Path", config);
+        String group = config.getConfig().getFirst(GroupMapperConfig.LDAP_GROUPS_PATH).trim();
+        if (!"/".equals(group) && KeycloakModelUtils.findGroupByPath(realm, group) == null) {
+            throw new ComponentValidationException("Groups path group '" + group + "' does not exist - please create it first");
```

Comment:
Is it possible to use generic message without adding "group" variable into the text? Maybe just something like "Groups path group does not exist  - please create the group on specified path first" .

The reason is some potential for various injection attacks, which would be possible if the user input (variable "group") is shown in the notification in the UI.

Bonus point: If you manage to localize the error message, it will be nice as well. The message with specified key will just need to be added to this file and that should be enough: https://github.com/keycloak/keycloak/blob/master/themes/src/main/resources/theme/base/admin/messages/messages_en.properties

Reply:
Sure. And now I know why the other exception messages are so generic.

Reply:
And here we go